src/components/global/GNB.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+import THEME from 'constants/Colors';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { Mypage, TodoBox, TodoMall } from './Icons';
+
+type Props = {};
+
+const GNB = (props: Props) => {
+  const router = useRouter();
```

Comment:
사용하는 메서드만 뽑아서 사용하는 것은 어떨까요? 가독성에 더 좋아보여요!

```suggestion
const { push, asPath }  = useRouter();
```